Sources/iron/object/Object.hx

```diff
@@ -120,6 +123,12 @@ class Object {
 		traits.remove(t);
 	}
 
+	/**
+	 * Returns the instance of the Trait attached to the Game Object. 
+	 *
+	 * @param	Trait - The class name of the Trait instance to retrieve.
```

Comment:
This should be `c` as that is the argument name. `Trait` is the [type parameter constraint](https://haxe.org/manual/type-system-type-parameter-constraints.html). Also `class name` reads (at least for me) like it would be a string, but is in fact the class. 

So it should read more like `@param c - The class of the Trait instance to retrieve.`

Reply:
ah yes the Object class has a name:String property. I can see how that would be confusing to people.